framework/sandstone.cpp

```diff
@@ -3762,7 +3758,10 @@ int main(int argc, char **argv)
             }
         }
 
-        lastTestResult = run_one_test(*it, per_cpu_failures);
+        if (it->test->quality_level >= sApp->requested_quality)
+            lastTestResult = run_one_test(*it, per_cpu_failures);
+        else
+            logging_printf(LOG_LEVEL_QUIET, "Test %s is in BETA quality. Try again with --beta option.\n", it->test->id);
```

Comment:
Printing it like this will break the YAML output. Instead, move the checking into `run_one_test` and do a `log_skip()` like we do for the CPU feature limit.

Reply:
The CI caught this issue.

Reply:
It also should not be `LOG_LEVEL_QUIET`. At least on one `-v`.